pkgs/applications/office/timekeeper/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchFromGitHub
+, poco
+, pkgconfig
+, gnome2
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "timekeeper";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "bburdette";
+    repo = "TimeKeeper";
+    rev = "363a3f0033fcca2d45b0d1f0fc74d6470c9db10d";
+    fetchSubmodules = true;
+    sha256 = "03alrfy7axq64k9lrs14afc0m6zn5gr5glhgx9i2r1yam7wdmz5c";
+  };
+
+  buildInputs = [
+    poco
+    pkgconfig
```

Comment:
In new expressions, `pkg-config` should be used instead of the `pkgconfig` alias. Also it is a build tool so it should go to `nativeBuildInputs`.

Reply:
I did this:

```
  nativeBuildInputs = [
    poco
    pkg-config
  ];
```

But it says

```
bburdette@BB-5520:~/op-code/nixpkgs$ nix-env -f . -iA timekeeper
error: undefined variable 'pkg-config' at /home/bburdette/op-code/nixpkgs/pkgs/applications/office/timekeeper/default.nix:23:5
```

Reply:
Do not forget to also change the function arguments at the top of the file. Same for gtkmm2.

Reply:
ugh, of course!  thx.